emails/2023-03-17.md

```diff
@@ -0,0 +1,29 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+This week, my co-founder [Aaron](https://www.dolthub.com/team#aaron) let me in on a long standing joke. We went through a brief stint a year ago, trying to do video marketing. You can see the evidence on [our YouTube channel](https://www.youtube.com/@dolthub4295). We had a film crew come by and film some B-roll and they used some of it in the videos. When I was reviewing one of these I was confronted with this gem.
+
+[![Aaron playing Cookie Clicker](../images/aaron-cookie-clicker.png)](https://www.youtube.com/watch?v=cMuMsPWRuCU&t=406s)
+
+I got all mad and I was like, "You can't be playing Cookie Clicker in our videos!" He told me this week that it was staged just to mess with me. Well played, sir. Well played.
+
+### Versioned MySQL Replica
+
+We settled on a name for the new MySQL binlog replication mode I've ben talking about a lot: the Versioned MySQL Replica. This week we published [a Getting Started guide for this mode](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/). 
+
+[![Versioned MySQL Replica](../images/versioned-mysql-replica.png)](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/)
+
+In this mode, you set up Dolt as a replica of your primary MySQL database. Every write to the primary is replicated to Dolt and a new Dolt commit is created. This means you get the ability to find bad changes to your primary MySQL and create SQL patches to roll bad changes back. [The blog](https://www.dolthub.com/blog/2023-03-15-getting-started-versioned-mysql-replica/) has detailed example. You can do all this without migrating your primary off of MySQL. Just add Dolt.
+
+### Dolt 0.75
+
+Just in case you missed last week, we turned the chunk journal on by default making [the current Dolt release](https://github.com/dolthub/dolt/releases/tag/v0.75.3) ACID compliant. Upgrade now. Beyond durability it is also faster. We are down to [1.9X MySQL on sysbench benchmarks](https://docs.dolthub.com/sql-reference/benchmarks/latency).
```

Comment:
Should we start saying we are 90% slower than MySQL? Or wait until we are more like <50%.

Reply:
I've had the same thought...

I think 90% slower sounds worse than 2X